app/views/submissions/index.html.haml

```diff
@@ -35,11 +36,15 @@
               = link_to submission.corresponding_author, submission.author
             - else
               = submission.corresponding_author
+
           %td
             - if submission.person
               = link_to submission.editor, submission.person
             - else
               = submission.editor
+          %td
```

Comment:
Usunąłbym to, ponieważ w tej tabelce jest już b. dużo informacji. Natomiast wyświetliłbym to w widoku zgłoszenia.